cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   containers:
   - name: fluentd-elasticsearch
-    image: gcr.io/google_containers/fluentd-elasticsearch:1.6
+    image: satnam6502/elasticsearch:1.7
```

Comment:
I could be wrong here, but shouldnt you have updated cluster/addons/fluentd-elasticsearch/es-controller.yaml instead of this file.
This file uses "gcr.io/google_containers/fluentd-elasticsearch:1.6" and not "gcr.io/google_containers/elasticsearch:1.5"